cf_xarray/accessor.py

```diff
@@ -372,6 +372,7 @@ def _get_all(obj: Union[DataArray, Dataset], key: str) -> List[str]:
     """
     all_mappers = (
         _get_custom_criteria,
+        functools.partial(_get_custom_criteria, criteria=cf_role_criteria),
```

Comment:
@kthyng This was really easy!

Reply:
@dcherian Nice! Not sure what this is exactly, but glad the changes are working for you.